chart/values.yaml

```diff
@@ -0,0 +1,234 @@
+# Default values for kube-httpcache.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: quay.io/mittwald/kube-httpcache
+  pullPolicy: IfNotPresent
+  tag: "stable"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# Enable StatefulSet (Deployment is default)
+useStatefulset:
+  enabled: true
+
+# Enable configMap for Varnish Template File (see below vclTemplate)
+# OR use extravolume with name "template" if the file is too big
+configmap:
+  enabled: true
+
+# kube-httpcache specific configuration
+cache:
+  # name of backend service
+  backendService: backend-service
+  # name of backend service namespace
+  # backendServiceNamespace: backend-service-namespace
```

Comment:
if this is used because your backend is on a different namespace from kube-httpcache.. the rbac defined in the chart is not enough to let the operator work...

as per issue #40 a clusterrolebinding is needed..

```
kubectl create clusterrolebinding kube-httpcache --clusterrole=kube-httpcache --serviceaccount=[NAMESPACE_HERE]:kube-httpcache
```

Reply:
well also the `clusterrole`

Reply:
Either that, or alternatively, a `ClusterRole` and two separate `RoleBindings` for each namespace might also do the trick. Although it might be surprising (as in "principle of least surprise") for a user when the Helm chart starts creating new RoleBindings in namespaces other than the release namespace.

Reply:
This is the last remaining discussion on this PR, and I don't have a strong opinion yet on how to best proceed. Maybe the easiest solution would be to test for `.Values.cache.backendServiceNamespace` in the `NOTES.txt` template and notify the user to create the required role bindings themselves.

Any thoughts on this? /cc @blackjid @hensur

Reply:
mm, I think that the NOTES.txt idea is a good first approach.
Later we could add the roles and rolebindings... and cluster and clusterrolenbinding by setting some options in the values.yaml file

Reply:
:+1: Sounds good to me. Will adjust the NOTES.txt in a follow-up PR.